### PR TITLE
Update katharine.txt

### PR DESCRIPTION
**Pull request by jneubrand**
In this episode: Grammer, homosexuality, windows, trolls, printers, and a whole lot more.

### CODE REVIEW

File: kattrack/katharine.txt [hunk 1]
```diff
@@ -73,3 +73,41 @@ The amount of email I receive in my dreams is distressing.
...
+You know, that page that was linked to already did check that.
+The user can't not be logged in.
+Returning an empty string will break all sorts of things because that can never happen.
+Shiny.
+But how gay is it?
+Just windows being windows.
+But our role is to say "what the f***!?"
+WTF technically stands for "wise trolls faction"
+Actually, hm.
+I'm out at a thing.
```
Comment (thomasstoeckert) [inline kattrack/katharine.txt:84]: Maybe get rid of 84 and 85, not too interesting

File: kattrack/katharine.txt [hunk 2]
```diff
@@ -73,3 +73,41 @@ The amount of email I receive in my dreams is distressing.
...
+My timezone is lies.
+I don't know why it's started failing so often.
+Oxford comma: fight!
+So where do I complain about typos?
+Go complain to those who broke it.
+I don't understand this conversation.
+You seem to be arguing about something but I have no idea what.
+I can beautifully track my inability to sleep!
```
Comment (thomasstoeckert) [inline kattrack/katharine.txt:92]: We already have a quote like 93

File: kattrack/katharine.txt [hunk 3]
```diff
@@ -73,3 +73,41 @@ The amount of email I receive in my dreams is distressing.
...
+I think so?
+Also, we already have lawyers.
+Given infinite time and money, yes.
+I can't say anything about the future.
```
Comment (thomasstoeckert) [inline kattrack/katharine.txt]: Remove 94 through 97, they're kinda vague and boring

File: kattrack/katharine.txt [hunk 4]
```diff
@@ -73,3 +73,41 @@ The amount of email I receive in my dreams is distressing.
...
+Most people don't have issues. We know some do.
+It’s even more braindead than that!
+"Please fix my code" is generally a suboptimal request.
+Embrace the posix!
+And that's why new pebbles are better. They come in rose gold.
+Buy ALL THE WATCHES.
+These people are crazy.
+Which makes me want to bang my head against a wall.
+They were like "THAT TOOK ME TWO WEEKS HOW DID YOU DO THAT"
+OS X makes printers easy. :P
```
Comment (thomasstoeckert) [inline kattrack/katharine.txt]: Remove for boring

File: kattrack/katharine.txt [hunk 5]
```diff
@@ -73,3 +73,41 @@ The amount of email I receive in my dreams is distressing.
...
+I don't use facebook except when I do.
+[...] I already use twitter and I never figured out what I'm supposed to do with two social networks.
+Windows makes me sad. Windows running on awful hardware makes me really sad.
```
Comment (thomasstoeckert) [inline kattrack/katharine.txt:105]: Remove for obvious anti-windows bias :P

File: kattrack/katharine.txt (hunk 3 above)
Comment (jneubrand) [inline kattrack/katharine.txt]: won't remove 96, but otherwise 👌

File: kattrack/katharine.txt [hunk 6]
```diff
@@ -73,3 +73,41 @@ The amount of email I receive in my dreams is distressing.
...
+Learn everything! And do it while you’re young and have time to do so.
+Keyboard for scale!
+I also wonder.
```
Comment (thomasstoeckert) [inline kattrack/katharine.txt]: Remove 112& 113 for bore

File: kattrack/katharine.txt (hunk 5 above)
Comment (jneubrand) [inline kattrack/katharine.txt:105]: @thomasstoeckert no

File: kattrack/katharine.txt (hunk 2 above)
Comment (jneubrand) [inline kattrack/katharine.txt:92]: I think that's OK. this one is kinda funny and given the increased database size it shouldn't be that big of a deal

**Pull request merged**